adoc/chapters/programming_interface.adoc

```diff
@@ -4917,6 +4917,9 @@ info::event_command_status::complete
 ----
    a@ Indicates that the command has finished running on the device.
       Attempting to wait on such an event will not block.
+      
+      Synchronization: When [code]#event::get_info# returns this value,
+      the synchronization is equivalent to [code]#event::wait#.
```

Comment:
```suggestion
Synchronization: When [code]#event::get_info# returns this value,
the synchronization is equivalent to [code]#event::wait#.
```

Sorry, one more comment ... The indentation needs to be fixed like this, otherwise it gets rendered as a code block.

Reply:
Oh, I noticed that `reflow.py` didn't seem to work with the table. I should have double-checked the PDF. Thanks a lot!